activerecord/lib/active_record/persistence.rb

```diff
@@ -345,7 +345,7 @@ def increment(attribute, by = 1)
     def increment!(attribute, by = 1)
```

Comment:
Needs to collect the touch argument from the user and pass it on like this:

```ruby
# Wrapper around #increment that writes the update to the database.
# Only +attribute+ is updated; the record itself is not saved.
# This means that any other modified attributes will still be dirty.
# Validations and callbacks are skipped. Supports the `touch` option from
# +update_counters+, see that for more.
# Returns +self+.
def increment!(attribute, by = 1, touch: nil)
  # …
  self.class.update_counters(id, attribute => change, touch: touch)
  # …
end
```

Same equivalent for `decrement!` and a test there.